pkg/scheduler/scheduler.go

```diff
@@ -160,14 +162,85 @@ func (pc *Scheduler) loadSchedulerConf() {
 	}
 
 	pc.mutex.Lock()
-	// If it is valid, use the new configuration
+	for _, configuration := range configurations {
+		if configuration.Name == "dynamicConf" {
+			if err := pc.setConfKlogLevel(configuration); err != nil {
+				klog.Error(err)
+			}
+			if err := pc.setConfPercentageNodesToFind(configuration); err != nil {
+				klog.Error(err)
+			}
+			if err := pc.setConfSchedulePeriodDuration(configuration); err != nil {
```

Comment:
schedule period change will not take effect.

Reply:
yes,https://github.com/volcano-sh/volcano/blob/master/pkg/scheduler/scheduler.go#L95
```
// Run runs the Scheduler
func (pc *Scheduler) Run(stopCh <-chan struct{}) {
	pc.loadSchedulerConf()
	go pc.watchSchedulerConf(stopCh)
	// Start cache for policy.
	pc.cache.SetMetricsConf(pc.metricsConf)
	pc.cache.Run(stopCh)
	pc.cache.WaitForCacheSync(stopCh)
	klog.V(2).Infof("scheduler completes Initialization and start to run")
	go wait.Until(pc.runOnce, pc.schedulePeriod, stopCh)
	if options.ServerOpts.EnableCacheDumper {
		pc.dumper.ListenForSignal(stopCh)
	}
}
```
I need to rerun `go wait.Until(pc.runOnce, pc.schedulePeriod, stopCh)`to effect schedulePeriod,I don't know how to stop the `go wait.Until` and rerun it. @lowang-bh What is your suggestion？

Reply:
I don't know how to effect SchedulePeriodDuration.So i have deleted dynamic conf it.

Reply:
> What is your suggestion？

Yes, we don't need this one.

What's more, how about put a `Configuration` with name `feasibleConf` and keys `minNodesToFind`, `percentageOfNodesToFind` and  `minPercentageOfNodesToFind` in Configurations ?

Reply:
But it is strange both options and configuration has those parameters.